discovery-provider/src/tasks/entity_manager/user.py

```diff
@@ -133,3 +149,31 @@ def update_user(params: ManageEntityParameters):
     params.add_user_record(user_id, user_record)
 
     return user_record
+
+
+def verify_user(params: ManageEntityParameters):
+    validate_user_tx(params)
+
+    user_id = params.entity_id
+    existing_user = params.existing_records[EntityType.USER][user_id]
+    existing_user.is_current = False  # invalidate
+    if (
+        user_id in params.new_records[EntityType.USER]
+        and params.new_records[EntityType.USER][user_id]
+    ):  # override with last updated user is in this block
+        existing_user = params.new_records[EntityType.USER][user_id][-1]
```

Comment:
i actually don't think this is necessary either because add_user_record does this override

Reply:
noice